rtengine/helpersse2.h

```diff
@@ -117,19 +117,22 @@ static INLINE vfloat vcast_vf_f(float f)
 
 static INLINE vfloat vaddf(vfloat x, vfloat y)
 {
-    return _mm_add_ps(x, y);
```

Comment:
Yes, compiler (at least gcc) emits better code when not using intrinsics. For example:
_mm_add_ps(_mm_mul_ps(a,b),c) results in exactly these instructions, whereas a*b+c is compiled to an FMA instruction when compiled for chips with that feature.
I'll add some comments.